estate/util.py

```diff
@@ -0,0 +1,8 @@
+def action(function):
+    """When nothing is returned, return `True` as a default instead"""
+    def action_wrapper(*args, **kwargs):
+        result = function(*args, **kwargs)
+        if result == None:
```

Comment:
https://discuss.python.org/t/why-is-it-better-to-write-if-a-is-none-instead-of-if-a-none/42203
https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none
```suggestion
        if result is None:
```